packages/components/src/styles/components/advanced-table.scss

```diff
@@ -433,9 +433,15 @@ $hds-advanced-table-drag-preview-background-color: rgba(204, 227, 254, 30%);
     box-shadow: var(--token-elevation-low-box-shadow);
   }
 
+  // Use pseudo element for focus ring to allow offset position to be partially inside the button
```

Comment:
Use of pseudo-elements for focus on AdvancedTable child components can be re-examined at a later time, as it seems to necessitate visual design changes.